src/components/Table/utils/utils.tsx

```diff
@@ -102,7 +102,7 @@ export function toContent(
         {tooltipEl}
       </>
     );
-  } else if (style.emptyCell && isContentEmpty(content)) {
+  } else if (!isHeader && !isExpandableHeader && style.emptyCell && isContentEmpty(content)) {
```

Comment:
It seems like it'd be odd for an empty header and expandable header to show the `emptyCell` style. It makes more sense in a `totals` row, or in cell within the table's body. Hopefully this eliminates the need to remember to use `header: emptyCell`.

Reply:
That shouldn't be the case if the value is actually `0`, especially if the value is `$0` (a string). I would guess that is happening locally. GridTable only shows the empty cell dash when it is an empty string, null, or undefined.

This is the code that is used to determine it:
```
const emptyValues = ["", null, undefined] as any[];

function isContentEmpty(content: ReactNode): boolean {
  return emptyValues.includes(content);
}
```

Reply:
Ah! That makes sense, thanks! Yep, I see where we're explicitly passing `undefined` instead of `0`. Will fix it there.